src/components/structures/SpaceHierarchy.tsx

```diff
@@ -63,7 +63,13 @@ interface IProps {
     space: Room;
     initialText?: string;
     additionalButtons?: ReactNode;
-    showRoom(cli: MatrixClient, hierarchy: RoomHierarchy, roomId: string, autoJoin?: boolean): void;
+    showRoom(
+        cli: MatrixClient,
+        hierarchy: RoomHierarchy,
+        roomId: string,
+        autoJoin?: boolean,
+        isSpaceRoom?: boolean,
```

Comment:
we might want to pass through the room type instead of a boolean for future cases, but that's not super important right now

Reply:
good point, will change